src/helpers.ts

```diff
@@ -6,14 +9,32 @@ export function assertUnreachable(_never: never): never {
 	throw new Error('Switch validation failed, look for assertUnreachable(...)')
 }
 
-function isValidDate(d: unknown) {
-	return d instanceof Date
+function isValidDate(d: Date) {
+	return !isNaN(d.getTime())
+}
+
+export function ParseModifiedDateFromInewsStoryWithFallbackToNow(story: INewsStory) {
```

Comment:
Minor detail. I suggest that we use camelCase, instead of PascalCase, for function names:
```suggestion
export function parseModifiedDateFromInewsStoryWithFallbackToNow(story: INewsStory) {
```

Reply:
As well as adding return types for functions.
```suggestion
export function ParseModifiedDateFromInewsStoryWithFallbackToNow(story: INewsStory): Date {
```

Reply:
Sure. Strange that the linter doesn't enforce the casing